gematria/model/python/token_model_flags.py

```diff
@@ -38,6 +37,18 @@
     ),
 )
 
+_ANNOTATION_NAME_FILE = flags.DEFINE_string(
```

Comment:
Very nitpicky: `_ANNOTATION_NAMES_FILE` to match the name of the command-line flag.